config/sync/field.field.user.user.field_user_frequency.yml

```diff
@@ -14,7 +14,7 @@ field_name: field_user_frequency
 entity_type: user
 bundle: user
 label: 'Notification frequency'
-description: 'How often do you prefer to receive status updates on new and/or updated content on Joinup?'
+description: 'How often do you prefer to receive status updates on new and/or updated content?'
```

Comment:
There is a copy of this config file in `web/modules/custom/joinup_subscription/config/install/field.field.user.user.field_user_frequency.yml`. Please, update that file accordingly.